internal/provider/helpers/compare.go

```diff
@@ -1,14 +0,0 @@
-package helpers
-
-import "github.com/go-test/deep"
-
-// ObjectsEqual checks to see if two objects are equivalent, accounting for
-// differences in the order of the contents.
-func ObjectsEqual(obj1, obj2 interface{}) (bool, []string) {
```

Comment:
This is replaced by `NormalizedValueForJSON`, where we rely on the `jsontypes.Normalized` type to calculate equality.